src/translations/en/common.json

```diff
@@ -71,13 +74,5 @@
   "widget.action.confirm": "Confirm",
   "widget.action.download-chat": "Download conversation",
   "widget.action.forward-chat": "Forward to an email",
-  "widget.action.forward-email": "Your email address",
-  "widget.form.phone": "Your phone number",
-  "widget.form.email": "Your email address",
```

Comment:
Please revert waiting time removed translations as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445